### PR TITLE
Keys.ResetCache fix

### PR DESCRIPTION
**Pull request by Backs**

### CODE REVIEW

File: src/Statiq.Web.Hosting/Middleware/DefaultExtensionsMiddleware.cs [hunk 1]
```diff
@@ -35,7 +35,7 @@ public DefaultExtensionsMiddleware(RequestDelegate next, IWebHostEnvironment hos
 
             _next = next ?? throw new ArgumentNullException(nameof(next));
             _fileProvider = hostingEnv.WebRootFileProvider;
-            _extensions = (options?.Value ?? new DefaultExtensionsOptions()).Extensions.Select(x => x.StartsWith(".") ? x : ("." + x)).ToArray();
+            _extensions = (options.Value ?? new DefaultExtensionsOptions()).Extensions.Select(x => x.StartsWith(".") ? x : ("." + x)).ToArray();
```
Comment (Backs) [inline src/Statiq.Web.Hosting/Middleware/DefaultExtensionsMiddleware.cs:38]: `options` not null

File: src/Statiq.Web.Hosting/Middleware/VirtualDirectoryMiddleware.cs [hunk 2]
```diff
@@ -28,7 +28,7 @@ public async Task InvokeAsync(HttpContext context)
             if (context.Request.Path.ToString().StartsWith(_virtualDirectory))
             {
                 string realPath = context.Request.Path.ToString().Substring(_virtualDirectory.Length);
-                if (realPath?.Length == 0)
+                if (realPath.Length == 0)
```
Comment (Backs) [inline src/Statiq.Web.Hosting/Middleware/VirtualDirectoryMiddleware.cs:31]: `realPath` not null

File: src/Statiq.Web/Commands/PreviewCommand.cs [hunk 3]
```diff
@@ -149,6 +149,7 @@ internal class PreviewCommand : PipelinesCommand<PreviewCommandSettings>
                         {
                             ConfigurationSettings[Keys.ResetCache] = false;
                         }
+                        else
```
Comment (Backs) [inline src/Statiq.Web/Commands/PreviewCommand.cs:152]: missing

**Comment by daveaglick** [comment]
Awesome, thanks! Somehow totally missed this PR - glad to get these fixes merged, the kind of stuff that would eventually bite someone.

**Pull request merged**